src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java

```diff
@@ -725,12 +715,18 @@ public Coding tRoleCode2PatientContactRelationshipCode(String cdaRoleCode) {
 			case "stpprn": // step parent
 			case "stpfth": // stepfather
 			case "stpmth": // stepmother
-				code = "parent"; display = "Parent"; break;
-			case "powatt":
-				code = "agent"; display = "Agent"; break;
+				code="C"; display = "Emergency Contact"; break;
+			case "work": 
+				code = "E"; display = "Employer"; break;
+			case "gt": 
+				code = "BP"; display = "Billing contact person"; break;
+			case "fammemb":
```

Comment:
It looks like BP is deprecated and removed.

Reply:
I'll change it to other.